OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -56,8 +56,12 @@ public class CargoInfo : ITraitInfo, Requires<IOccupySpaceInfo>
 		public readonly string UnloadBlockedCursor = "deploy-blocked";
 
 		[UpgradeGrantedReference]
-		[Desc("The upgrades to grant to self while loading cargo.")]
-		public readonly string[] LoadingUpgrades = { };
+		[Desc("The condition to grant to self while waiting for cargo to load.")]
+		public readonly string LoadingCondition = null;
+
+		[UpgradeGrantedReference]
+		[Desc("Conditions to grant when specified actors are loaded inside the transport.")]
```

Comment:
Please explain that the first string should be an actor type and the second the condition.